helix-core/src/main/java/org/apache/helix/GroupCommit.java

```diff
@@ -112,6 +112,9 @@ public boolean commit(BaseDataAccessor<ZNRecord> accessor, int options, String k
             merged = accessor.get(mergedKey, null, options);
           } catch (ZkNoNodeException e) {
             // OK.
+          } catch (Exception e) {
```

Comment:
There are a few places that call DataAccessor.updateProperty but did not check the return results, we may want to double check whether it is appropriate (i.e, adding check there if it cares writing failure).

one example is:  HelixAdmin->addTypeToCustomizedStateConfig()

Reply:
Yep, I'll address them together with #1436

Reply:
Can you search all existing use cases (no related to customized state) and make sure there is no assumption on old behavior?

Reply:
This change only impacts group commit. As shown in the following, only current state and customized state use group commit. 
case CURRENTSTATES:
case CUSTOMIZEDSTATES:
        success = _groupCommit.commit(_baseDataAccessor, options, path, value.getRecord(), true);

However, as you mentioned, for normal update, there're cases that don't really check response. I'll take a look to see whether they're reasonable.

Reply:
Done. Actually no impact on other function.